packages/cli-types/src/index.ts

```diff
@@ -127,7 +128,7 @@ export type ProjectConfig = {
  * @property platforms - Map of available platforms (build-ins and dynamically loaded)
  * @property commands - An array of commands that are present in 3rd party packages
  */
-export type Config = {
+export interface Config extends IOSNativeModulesConfig {
```

Comment:
This is done purely to ensure the `Config` doesn’t deviate in a way that the otherwise uncheckable Ruby source.